api/src/main/java/org/pmiops/workbench/access/AccessUtils.java

```diff
@@ -76,4 +77,25 @@ public static DbAccessTier getAccessTierByShortNameOrThrow(
         .orElseThrow(
             () -> new NotFoundException("Access tier " + accessTierShortName + "not found"));
   }
+
+  // missing ERA_COMMONS (special-cased)
+  // see also: AccessModuleServiceImpl.isModuleRequiredInEnvironment()
+  public static final List<AccessModuleName> REQUIRED_MODULES_FOR_REGISTERED_TIER =
+      ImmutableList.of(
+          AccessModuleName.DATA_USER_CODE_OF_CONDUCT,
+          AccessModuleName.PROFILE_CONFIRMATION,
+          AccessModuleName.PUBLICATION_CONFIRMATION,
+          AccessModuleName.RAS_LOGIN_GOV,
+          AccessModuleName.RT_COMPLIANCE_TRAINING,
+          AccessModuleName.TWO_FACTOR_AUTH);
+
+  public static final List<AccessModuleName> REQUIRED_MODULES_FOR_CONTROLLED_TIER =
+      ImmutableList.of(
+          AccessModuleName.CT_COMPLIANCE_TRAINING,
+          AccessModuleName.DATA_USER_CODE_OF_CONDUCT,
```

Comment:
i was just thinking would it make sense to have a list like REQUIRED_MODULES = AccessModuleName.PROFILE_CONFIRMATION,
          AccessModuleName.PUBLICATION_CONFIRMATION,
          AccessModuleName.RAS_LOGIN_GOV,
          AccessModuleName.RT_COMPLIANCE_TRAINING,
          AccessModuleName.TWO_FACTOR_AUTH
          
          and then RT and CT list can inherit that and add their own modules (eg           AccessModuleName.CT_COMPLIANCE_TRAINING ) 
          This way it will be easier to read which module is different for RT and CT